### DISCUSSION TITLE
Doppelt als Service installieren?

### DISCUSSION THREAD

**Discussion by Cr4Pi** [Schrolli91/BOSWatch#529] [General]
Hi folks,

wie kann ich boswatch doppelt als service installieren?

Muss ich den Boswatch Ordner kopieren?
Welche Dateien muss ich bearbeiten?

Als einfacher Service läuft er ohne Probleme... mit der Option -d komme ich auch zurecht
aber beim "doppelten Service" bin ich irgendwie raus...

Danke für eure Unterstützung!

Schönes Wochenende noch...

**Comment by flothi** [comment]
Verständnisfrage: Hast du zwei sdr-Sticks oder willst du mit zwei Instanzen auf einen zugreifen? Du könntest es wahrscheinlich damit lösen, dass du zweimal einen Service instanziierst, aber da müsste die config.ini bei beiden gleich sein. Andernfalls bleibt nur die "stumpfe" Kopie.

**Comment by Cr4Pi** [comment]
der Plan ist 2 SDR-Sticks an einem Pi laufen zu lassen und beides als service installieren.
